### ISSUE TITLE
Wordpress

### ISSUE THREAD

**Issue by ghost** [adgsm/multi-level-push-menu#57]
I've been having problems integrating this into wordpress. Is this even possible?

**Comment by Peter-Boardman** [comment]
Did you ever get this to work in WP?

**Comment by judia** [comment]
No. I gave it up. Wordpress doesn't like it for whatever reason.

**Comment by Peter-Boardman** [comment]
Hi,

Thank you for your quick response. Are you the developer of *GT Push Menu
Lite* from Griffin Themes? If so, I have the the *lite* (free) version of
the menu plugin working in WordPress now. I'd like to purchase the PRO
version to use it on a site that I'm working on.

Is there a way to purchase the Pro version at all?

Or is the pro version completely broken?

Thanks very much, I look forward to your response.

Cheers,

kind regards,

Peter Boardman
Web Developer, Art Director

[tel]                     (416) 565-2422
[web]                  peterboardman.ca
[twitter]               @peterboardmanca
<http://twitter.com/peterboardmanca>
[skype]                peterboardman.ca

Development & Design  >>  Website | Kiosk | eLearning | Animation

HTML5, CSS3, jQuery, JavaScript, Node, Git, Gulp, Grunt, Bourbon, Neat,
SASS, LESS, Bootstrap, Foundation, Flash, AIR, Edge, CMS: Joomla &
WordPress, MVC JavaScript Frameworks, PHP, MySQL, Apache, JSON, XML & more…
Websites | e-Learning | Kiosks | Flash & HTML5 Games | Mobile & Desktop
Applications | Broadcast & Corporate Animation | Speaker Support
Presentations | Project Management

**Comment by judia** [comment]
I'm not sure if that was directed at me or not. I was trying to integrate it into a theme, rather than a plugin.
This isn't my project, though. I just found it by accident. It works great as a plugin, but that's not what I was aiming for.

**Comment by lastgeniusleft2004** [comment]
hey judia,
we are a team of 10 who has developers and testers and we do mobile app development aswell ,
you can check out our website www.whitefairapps.com.
we are based in windermere florida you can mail me at shawn@whitefairapps.com and please let me know your concerns in the mail please.

Thanks,
Shawn-

**Comment by GASQUY** [comment]
Hi, 

I create theme wordpress with multi level push menu ;) [multi level push menu](http://www.angeldisc.com/)

Good day
